README.md

```diff
@@ -1,9 +1,10 @@
-# Name Request Online Frontend
+# Name Request UI
 
 [![img](https://img.shields.io/badge/Lifecycle-Stable-97ca00)](https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md)
 
 ## Project setup
 ```
+npm install -g pnpm
 pnpm install
```

Comment:
I had to look up how to use pnpm, so here I'm saving the next person a couple of minutes :)

Reply:
Same thing happened to me a while ago. Good idea.